modules/mesaides/templates/nginx_config.erb

```diff
@@ -0,0 +1,56 @@
+# MANAGED BY PUPPET
+server {
+  listen                *:80 default_server;
+  server_name           <%= @name %>;
+
+  access_log            /var/log/nginx/<%= @name %>.access.log combined;
+  error_log             /var/log/nginx/<%= @name %>.error.log;
+
+  location /.well-known {
+    root                <%= @webroot_path %>;
+  }
+<% if @use_ssl and File.exists?("/etc/letsencrypt/live/#{@name}/fullchain.pem") -%>
```

Comment:
I'd rather have an explicit failure if `use_ssl` is requested and the certificate is not provided. Either that, or use the fact that a certificate is provided as a sign that SSL should be used. But this silent failure is problematic.